vendor/assets/javascripts/bootstrap_toggle.js

```diff
@@ -176,4 +176,8 @@
     $checkbox.bootstrapToggle('toggle')
     e.preventDefault()
   })
+<<<<<<< HEAD:vendor/assets/javascripts/bootstrap_toggle.js
```

Comment:
Can you remove lines 179-182 here? Looks like a sketch artifact of a merge conflict plus an assets:precompile hash.